custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -22,6 +22,11 @@ def __init__(self, month):
         self.month_end_15yr = self.month_end - relativedelta(years=15)
         self.month_start_18yr = self.month_start - relativedelta(years=18)
 
+    @property
+    def child_temp_tablename(self):
+        from custom.icds_reports.utils.aggregation_helpers.distributed import ChildHealthMonthlyAggregationDistributedHelper
+        return ChildHealthMonthlyAggregationDistributedHelper([], self.month_start).temporary_tablename
```

Comment:
Would it be possible to extract this out into a function instead of instantiating the class?